.eslintrc.json

```diff
@@ -22,6 +22,7 @@
     ],
     "rules": {
         "react/react-in-jsx-scope": "off",
-        "@typescript-eslint/explicit-module-boundary-types": "off"
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "react/prop-types": "off"
```

Comment:
Likewise to my other comment, I normally wouldn't want this off but definitely easier to toggle this off for today :)